topics/event-classes.md

```diff
@@ -32,5 +32,6 @@ below:
 | `reviews.actions.setOutcome.1` | An acquisitions review outcome was set |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#reviews-actions-setOutcome-1)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```

Comment:
Also noticed that there are no docs for "list.edit" events, do we need to add it?

Reply:
We will.  One for me, I suspect.

Reply:
Ah - missed this. Do we also need to do the same for the digitisation events I've recently added? Not sure since they're timeseries and I don't know if we intend for them to be used by customers...